### DISCUSSION TITLE
Image on top of splash-pages is not getting displayed on GitHub Pages

### DISCUSSION THREAD

**Discussion by gitgilleszeimet** [thegetty/quire#444] [Get Help] [answered]
I want to use Quire for an upcoming project and in order to do so I am currently gaining experience by playing around with a default project (https://gitgilleszeimet.github.io/quire-test).

I have encountered the following problem:

The image on top of splash-pages (https://gitgilleszeimet.github.io/quire-test/intro/) is not getting displayed. Is this a bug or am I doing something wrong?

The above mentioned test case is an unmodified default project I have uploaded to GitHub Pages.

Thank you for your help,

Gilles

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @gitgilleszeimet. Thank you for bringing your question to the forum. I was actually experiencing this issue just the other day but didn't have a chance to dig deeper. What version of Quire are you on (you can determine this by running `quire --version` in your Terminal.)

**Reply by gitgilleszeimet** [reply]
Hi @Erin-Cecele . I run version v0.20.4.

**Reply by geealbers** [reply]
Hi @gitgilleszeimet, this issue can happen when deploying a Quire site via GitHub pages. To fix it go to Line 14 in `themes/default/layouts/splash/single.html`, and change `relURL` to `absURL` in this section near the end of the line: `style="background-image: url('{{ $imgPath | relURL }}');"`

So that full line will be:

```
<section class="{{ if or (eq .Title "title page") (eq .Title "half title page") }} is-screen-only {{ end }} quire-page__header hero {{ if .Content }}{{ else }}quire-page__header--full-height{{ end }} {{ if .Params.image }}hero-image{{ end }}" style="background-image: url('{{ $imgPath | absURL }}');">
```

**Reply by gitgilleszeimet** [reply]
Hi @geealbers, thank your for your help. In the meantime I have moved the publication to my own webspace (http://gilleszeimet.lu/mosaik-marmor-musen). Unfortunately, your solution has not solved the issue (http://gilleszeimet.lu/mosaik-marmor-musen/04-chap-1/).

Is there anything else I could so?

**Reply by geealbers** [reply]
In the `config/site.yml` file, try changing `baseURL: "/"` to `baseURL: "http://gilleszeimet.lu/mosaik-marmor-musen/"`, and then redeploy your site.

**Reply by gitgilleszeimet** [reply]
I changed the config/site.yml, but still no improvement.

**Reply by geealbers** [reply]
Hmmm. Can you explain your deploy method to me? Do you run `quire site` and then upload the resulting folders to your server? And can I confirm that the change from `relURL` to `absURL` in `themes/default/layouts/splash/single.html` is still in place?

**Reply by gitgilleszeimet** [reply]
Hi Greg! I will get back to you next week upon my return.

**Reply by gitgilleszeimet** [reply]
Hi @geealbers ! Yes, I run quire site and then I upload the content of the site folder the mosaik-marmor-musen folfer on my webserver via ftp. 

The line in themes/default/layouts/splash/single.html reads:

<section class="{{ if or (eq .Title "title page") (eq .Title "half title page") }} is-screen-only {{ end }} quire-page__header hero {{ if .Content }}{{ else }}quire-page__header--full-height{{ end }} {{ if .Params.image }}hero-image{{ end }}" style="background-image: url('{{ $imgPath | absURL }}');">

So, absURL appears do be in place.

Unfortunately, the image is not getting displayed.

**Reply by geealbers** [reply]
@gitgilleszeimet I think I may have figured it out. Can you check if you have a `config/_default/config.yml` file in your project? If you do, please delete it, and then try deploying your site again. (Still with the absURL change, and with `baseURL: "http://gilleszeimet.lu/mosaik-marmor-musen/"` set in `config/site.yml`.)

**Reply by gitgilleszeimet** [reply]
@geealbers Deleting `config/_default/config.yml` solved the problem! Thank you for your help.
